pyproject.toml

```
@@ -1,3 +1,41 @@
+[project]
+name = "spidev"
+description = "Python bindings for Linux SPI access through spidev"
+dynamic = ["readme"]
+version = "3.8"
+
+authors = [
+    { name = "Volker Thoms", email = "unconnected@gmx.de" },
+]
+maintainers = [
+    { name = "Stephen Caudle", email = "scaudle@doceme.com" },
+]
+license = "MIT"
+license-files = ["LICENSE"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Operating System :: POSIX :: Linux",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development",
+    "Topic :: System :: Hardware",
+    "Topic :: System :: Hardware :: Hardware Drivers",
+]
+keywords = ["SPI", "Linux", "SPI device"]
+
 [build-system]
 requires = ["setuptools>=61.0"]
```

With the other changes, we are now depending on a more recent version of `setuptools`, so this should be lifted to v80.

```suggestion
requires = ["setuptools>=80.0"]
```